pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -157,8 +157,9 @@ func newCertRotationController(
 			Name:      "aggregator-client-signer",
 			AdditionalAnnotations: certrotation.AdditionalAnnotations{
 				JiraComponent:                    "kube-apiserver",
-				AutoRegenerateAfterOfflineExpiry: "https://github.com/openshift/cluster-kube-apiserver-operator/pull/1631,'[sig-cli] oc adm new-project [apigroup:project.openshift.io][apigroup:authorization.openshift.io] [Suite:openshift/conformance/parallel]'",
```

Comment:
are there any other changes except moving the test name to the new field ?

Reply:
Nope, squashed two commits here (the second commit was fixing issues introduced by the first one, so no need to keep it around)